examples/How_to_stream_completions.ipynb

```diff
@@ -17,7 +17,7 @@
     "\n",
     "## Downsides\n",
     "\n",
-    "Note that using `stream=True` in a production application makes it more difficult to moderate the content of the completions, as partial completions may be more difficult to evaluate. which has implications for [approved usage](https://beta.openai.com/docs/usage-guidelines).\n",
+    "Note that using `stream=True` in a production application makes it more difficult to moderate the content of the completions, as partial completions may be more difficult to evaluate. Which has implications for [approved usage](https://beta.openai.com/docs/usage-guidelines).\n",
```

Comment:
```suggestion
    "Note that using `stream=True` in a production application makes it more difficult to moderate the content of the completions, as partial completions may be more difficult to evaluate. This may have implications for [approved usage](https://beta.openai.com/docs/usage-guidelines).\n",
```